detection-bot/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "forta-agent-nethermind-detector",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript nethermind detector project",
+  "chainIds": [
+    1
+  ],
```

```suggestion
  "name": "forta-bot-nethermind-detector",
  "version": "0.0.1",
  "description": "Forta Agent Typescript nethermind detector project",
  "chainIds": [
    1
  ],
```
I think the `"description"` could be improved to something like: "Forta bot that detects bot deployments from the Nethermind deployer address" or something like that. Feel free to only use my suggestion as a starting point.

`package.json` should also include a `"repository"` property, which we usually place following the `"chainIds"` property. The value would be the URL of where the repo would reside post-PR merge.

Lastly, the `"chainIds"` value should be updated to be the chain id for Polygon instead of mainnet Ethereum. (Can find a chain's chain id [here](https://chainlist.org/))